docs/launching-apps/localhost.rst

```diff
@@ -35,81 +35,3 @@ peers:
   and destination MPI processes can share memory (e.g., via SYSV or
   POSIX shared memory mechanisms).
 
```

Comment:
Could you leave a cross-reference here to the new page?  Perhaps something like:

```diff
diff --git i/docs/launching-apps/localhost.rst w/docs/launching-apps/localh>
index 37876952ae..be5a14c13d 100644
--- i/docs/launching-apps/localhost.rst
+++ w/docs/launching-apps/localhost.rst
@@ -35,3 +35,6 @@ peers:
   and destination MPI processes can share memory (e.g., via SYSV or
   POSIX shared memory mechanisms).
 
+  .. note:: For more information about using shared memory MPI
+            communication, see the :doc:`Shared Memory
+            </tuning-apps/networking/shared-memory>` page.
```

Reply:
Actually, in the interest of getting this merged, I took the liberty of adding this `note` and pushing it to this PR.